set.mm

```diff
@@ -475884,15 +476368,25 @@ The sumset operation can be used for both group (additive) operations and
     elringlsm.2 $e |- .x. = ( .r ` R ) $.
     elringlsm.3 $e |- G = ( mulGrp ` R ) $.
     elringlsm.4 $e |- .X. = ( LSSum ` G ) $.
-    elringlsm.5 $e |- ( ph -> G e. V ) $.
     elringlsm.6 $e |- ( ph -> E C_ B ) $.
     elringlsm.7 $e |- ( ph -> F C_ B ) $.
     $( Membership in a product of two subsets of a ring.  (Contributed by
        Thierry Arnoux, 20-Jan-2024.) $)
     elringlsm $p |- ( ph -> ( Z e. ( E .X. F )
                             <-> E. x e. E E. y e. F Z = ( x .x. y ) ) ) $=
-      ( wcel wss co cv wceq wrex wb mgpbas mgpplusg lsmelvalx syl3anc ) AJKTHDU
-      AIDUALHIGUBTLBUCCUCFUBUDCIUEBHUEUFQRSBCDFGHIJKLDEJOMUGEFJONUHPUIUJ $.
+      ( cvv wcel wss co cv wceq wrex wb fvexi mgpbas mgpplusg lsmelvalx mp3an2i
+      cmgp ) JRSAHDTIDTKHIGUASKBUBCUBFUAUCCIUDBHUDUEJEUKNUFPQBCDFGHIJRKDEJNLUGE
+      FJNMUHOUIUJ $.
```

Comment:
Why did the proof become longer?

Reply:
It did because I removed one hypothesis, the `( ph -> G e. V )` could be proven from ``G = ( mulGrp ` R )``, the addionnal `fvexi` increased the proof by a bit.

Reply:
OK, I understand.